cmd/relayer_exporter/relayer_exporter.go

```diff
@@ -56,33 +92,54 @@ func main() {
 	)
 
 	ctx := context.Background()
-	// TODO: Add a feature to refresh paths at configured interval
-	paths, err := cfg.IBCPaths(ctx)
-	if err != nil {
-		log.Fatal(err.Error())
-	}
+	registry := prometheus.NewRegistry()
 
-	rpcs, err := cfg.GetRPCsMap(paths)
-	if err != nil {
+	// Initial setup of collectors
+	if err := refreshCollectors(ctx, cfg, registry); err != nil {
 		log.Fatal(err.Error())
 	}
 
-	ibcCollector := collector.IBCCollector{
-		RPCs:  rpcs,
-		Paths: paths,
-	}
+	// Start periodic refresh in background
+	var wg sync.WaitGroup
 
-	balancesCollector := collector.WalletBalanceCollector{
-		RPCs:     rpcs,
-		Accounts: cfg.Accounts,
-	}
+	wg.Add(1)
+
+	go func() {
+		defer wg.Done()
+
+		ticker := time.NewTicker(*refreshInterval)
+
+		defer ticker.Stop()
+
+		for {
+			select {
+			case <-ctx.Done():
```

Comment:
we should create context.WithCancel at line 94 and ensure that the context is cancelled appropriately when the main program exits.